src/main/java/bio/terra/service/snapshotbuilder/utils/constants/ConditionOccurrence.java

```diff
@@ -1,6 +1,7 @@
 package bio.terra.service.snapshotbuilder.utils.constants;
 
-public final class ConditionOccurrence {
+public class ConditionOccurrence {
+
```

Comment:
why are these tables (ConditionOccurrence and ObservationOccurrence) not moved over to the tables directory as well? Separate question, If we still need these tables, what is the purpose of the new DomainOccurrence table? Do we need tables for each domain - procedure, device, etc?